dbms/src/Flash/Pipeline/Exec/PipelineExec.cpp

```diff
@@ -17,12 +17,28 @@
 
 namespace DB
 {
+void PipelineExec::execPrefix()
+{
+    sink_op->operatePrefix();
+    for (auto it = transform_ops.rbegin(); it != transform_ops.rend(); ++it)
+        (*it)->operatePrefix();
+    source_op->operatePrefix();
+}
+
+void PipelineExec::execSuffix()
```

Comment:
```suggestion
void PipelineExec::executeSuffix()
```
Align with `execute()`